### PR TITLE
refactor(SinginAgreement): code 

### PR DESCRIPTION
**Pull request by Crazy-MIXER**

### CODE REVIEW

**Comment by Crazy-MIXER** [comment]
- Move text to i18n
- Replace navigation hooks
- Change the imports to absolute

File: src/components/restore-agreement/restore-agreement.tsx [hunk 1]
```diff
@@ -45,21 +45,19 @@ export const RestoreAgreement = ({onDone, testID}: RestoreAgreementProps) => {
...
-      </Text>
+      <Text t4 center i18n={I18N.restoreAgreementTitle} style={styles.title} />
       <Text
         t11
         center
         style={styles.disclaimer}
-        color={getColor(Color.textBase2)}>
-        {getText(I18N.restoreAgreementText)}
-      </Text>
+        color={getColor(Color.textBase2)}
```
Comment (gHashTag) [inline src/components/restore-agreement/restore-agreement.tsx:53]:
```jsx
color={Color.textBase2}
```

**Review by gHashTag — CHANGES_REQUESTED** [review]
Fix comment

File: src/components/restore-agreement/restore-agreement.tsx (hunk 1 above)
Comment (Crazy-MIXER) [reply src/components/restore-agreement/restore-agreement.tsx]: fix it

**Pull request merged**